File: Core/Entities/Course.cs
using System.Collections.Generic;

namespace Core.Entities
{
    public class Course
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public int Price { get; set; }
        public string PictureUrl { get; set; }
        public string Description { get; set; }
        public ICollection<BasketItem> BasketItems { get; set; }
    }
}

File: mobile/lib/models/course.dart
class Course {
  int id;
  String name;
  int price;
  String pictureUrl;
  String description;

  Course({this.id, this.name, this.price, this.pictureUrl, this.description});

  Course.fromJson(Map<String, dynamic> json)
      : id = json['id'],
        name = json['name'],
        price = json['price'],
        pictureUrl = json['pictureUrl'],
        description = json['description'];

  Map<String, dynamic> toJson() => {
        'id': id,
        'name': name,
        'price': price,
        'pictureUrl': pictureUrl,
        'description': description
      };
}
